spec/mongo/collection_spec.rb

```diff
@@ -723,12 +731,43 @@
         }.to raise_exception(Mongo::Error::BulkWriteError)
       end
     end
+
+    context 'when collection has a validator', if: find_command_enabled? do
+
+      let(:result) do
+        collection_with_validator.insert_many([{ a: 1 }, { a: 2 }])
+      end
+
+      it 'inserts successfully when documents are valid' do
+        expect(result.inserted_count).to eq(2)
+      end
+
+      let(:result2) do
+        collection_with_validator.insert_many([{ x: 1 }, { x: 2 }])
+      end
+
+      it 'raises a BulkWriteError when documents are invalid' do
+        expect {
+          result2
+        }.to raise_exception(Mongo::Error::BulkWriteError)
+      end
+
+      let(:result3) do
+        collection_with_validator.insert_many(
+            [{ x: 1 }, { x: 2 }], :bypass_document_validation => true)
+      end
+
+      it 'inserts successfully when documents are invalid but bypass_document_validation is true' do
```

Comment:
then a context here 
'when bypass_document_validation is true'

then a test

it 'inserts successfully'